packages/odyssey-react/src/components/Title/Title.stories.tsx

```diff
@@ -24,9 +24,11 @@ export default {
   },
 };
 
-const Template: Story<Props> = ({ level, visualLevel, children }) => (
-  <Title level={level} visualLevel={visualLevel} children={children} />
-);
+const Template: Story<ComponentProps<typeof Title>> = ({
```

Comment:
Are we really moving to not publicly exposing props for components? `ComponentProps` appeared to be an escape hatch for cases where developers hadn't made a public api. Forcing downstream developers to get at our types this way feels cumbersome / like an antipattern.